lib/dependencies/sub-process.ts

```diff
@@ -12,6 +12,8 @@ function makeSpawnOptions(options?: ProcessOptions) {
   }
   if (options && options.env) {
```

Comment:
this `if` can be removed as well due to the `else`

Reply:
this `if` is still required as it is what sets the options with the provided env, the else however is not as `process.env` is default anyway, will remove the else